src/components/organisms/UiList/UiList.vue

```diff
@@ -11,6 +15,7 @@
       >
         <UiListItem
           v-bind="item"
+          class="focus-hidden"
```

Comment:
```suggestion
```
As I remember we shouldn't use this class outside focus directive. And I don't understand why we need this class here